### ISSUE TITLE
What happens to fantasy teams at the end of a season?

### ISSUE THREAD

**Issue by arthuroe** [arthuroe/fp#34]
Do we `delete/hide` all fantasy teams created by users and ask them to create a new team the next season?
Let's have the discussion here.

**Comment by serubirikenny** [comment]
Yes...each season you have to create a new team since players have left and others have joined....although Ideally they create new teams but keep in the same leagues....

**Comment by serubirikenny** [comment]
I think for record keeping purposes we'll have to store the final league standings at the end of the season....but yeah...new team each season....
